Balance/Table Cells/WalletTableViewCell.swift

```diff
@@ -37,13 +29,13 @@ class WalletTableViewCell: UITableViewCell {
         return label
     }()
     
-    private let containerView: UIView = {
+    internal let containerView: UIView = {
         let view = UIView()
         view.backgroundColor = .clear
         return view
     }()
     
-    private let backgroundImageView: UIImageView = {
+    internal let backgroundImageView: UIImageView = {
```

Comment:
`internal` is not correct here. It essentially does nothing when used outside of a framework as it acts the same as `public` or no modifier at all, `private` is the correct modifier as these are meant to be visible only to this class. Internal only makes sense when you want something to be public, but only inside of a framework.